exercises/edward_brown/ex01_basics/src/Punctuation.cpp

```diff
@@ -0,0 +1,16 @@
+#include "Punctuation.h"
+#include <algorithm>
+
+bool isPunctuation(char Candidate) {
+  auto const punctuation = std::string(".,?'\\\"!():");
```

Comment:
:+1:

Also, is there a reason for using the assignment operator rather than initialising explicitly? I've always used the latter to avoid any suggestion there could be a temporary copy (although I'm sure this would be optimised out anyway)